rules/rbe_repo.bzl

```diff
@@ -416,6 +436,10 @@ def _pull_image(ctx, docker_tool_path, image_name):
     _print_exec_results("pull image", result, fail_on_error = True)
     ctx.report_progress("image pulled.")
 
+    # Create a dummy file with the image name to enable testing
+    # if container was pulled
+    ctx.file("image_name", image_name, False)
```

Comment:
Write some message to the file to indicate this file is just to indicating if the container is pulled or not?

Reply:
done